test/e2e/main/main_test.go

```diff
@@ -83,7 +84,9 @@ func TestUpdateOperatorPassword(t *testing.T) {
 	spec.Name = name
 	// Register it
 	testKube.CreateInfinispan(spec, tutils.Namespace)
-	defer testKube.DeleteInfinispan(spec, tutils.SinglePodTimeout)
+	spec.Labels = map[string]string{"test-name": "TestUpdateOperatorPassword"}
```

Comment:
@durgeshanaokar please pay attention, that `t.Name` is a test name and wan't to be hardcoded

Reply:
Ack, will take care